### PR TITLE
Version 3.1

### PR DESCRIPTION
**Pull request by larsll**
Upgrade to Version 3.1 to align with the latest versions of the Bundle.

Changes:
* Ability to set RTF
* Discontinued Tournament
* Added Head-to-Model

### CODE REVIEW

File: defaults/dependencies.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
-    "master_version": "3.0",
+    "master_version": "3.1",
     "containers": {
-        "rl_coach": "3.0.5",
-        "robomaker": "3.0.6",
-        "sagemaker": "3.0.2"
+        "rl_coach": "3.1.0-dev",
+        "robomaker": "3.1.0-dev",
```
Comment (breadcentric) [inline defaults/dependencies.json:5]: I reckon these will change to non -dev versions?

**Review by breadcentric — APPROVED** [review]

File: scripts/training/prepare-config.py [hunk 2]
```diff
@@ -31,6 +31,9 @@
 config['TRAINING_JOB_ARN'] = 'arn:Dummy'
 
 # Car and training 
+config['BODY_SHELL_TYPE'] = os.environ.get('DR_CAR_BODY_SHELL_TYPE', 'deepracer')
+if config['BODY_SHELL_TYPE'] == 'deepracer':
```
Comment (jochem725) [inline scripts/training/prepare-config.py:35]: Why is this if statement necessary?

File: defaults/dependencies.json (hunk 1 above)
Comment (larsll) [reply defaults/dependencies.json:5]: Yes. Will be updated once the new images are available.

File: scripts/training/prepare-config.py (hunk 2 above)
Comment (larsll) [reply scripts/training/prepare-config.py:35]: Depending on if you have a normal shell or a custom shell you need different parameters. Sadly I have not been able to make the console 'pick up' on it.

**Pull request merged**